lrauv_description/models/tethys_equipped/model.sdf

```diff
@@ -111,7 +111,7 @@
       <plugin

```

Comment:
There are still two `1000` on this file:

```
lrauv_description/models/tethys_equipped/model.sdf:98:        <fluid_density>1000</fluid_density>
lrauv_description/models/tethys_equipped/model.sdf:156:        <ocean_density>1000</ocean_density>
```

Reply:
Yeah our thruster plugin/tethys plugin does weird things when we change the density. I think this has to do with our unit conversions. For now I'll add a comment saying that we haven't touched these to the SDF. I will open a PR upstream to take angular velocity as a command for the Thrusters this will eliminate this confusion.

Reply:
That's concerning. If we tune for stability with mismatched world parameters, then we're pushing these issues somewhere else. Even if we use angular velocity for the thruster, we should figure out what's wrong with the force control and fix that too, since it may affect other users.

Reply:
Agreed but we need to fix that in a separate PR not this PR.

Reply:
To add to it I think the issue lies in the way we do weird unit conversions in tethys comms plugin.

Reply:
> our thruster plugin/tethys plugin does weird things when we change the density

I found the bug that was causing that, it was a detail on `TethysComm`, see https://github.com/osrf/lrauv/pull/137